### ISSUE TITLE
iOS version

### ISSUE THREAD

**Issue by amiantos** [amiantos/pibar#9]
If PiBar proves popular I'd probably like to build an iOS version.

I know a lot of people use an iOS app to make it easier for family to enable/disable the pi-hole when they run into issues, so as a start I think keeping the main app interface very simple is going to be a good idea.

Other ideas:
- Shortcuts support
- Apple Watch support

Please chime in if you have suggestions or ideas on what could make an iOS version really next level.

**Comment by Coder84619** [comment]
Yes I'd like Apple watch support! Definitely do a cool Infograph complication.

**Comment by amiantos** [comment]
Thinking about how people have members of the family who may need to disable/enable Pi-hole from time to time, maybe the iOS version of PiBar should have a "Family Mode" that'll keep them out of the more technical parts of the app (like editing black/white lists and extended stats) and essentially just turns the app into a big on/off button for them. Sounds like a good idea?

**Comment by Coder84619** [comment]
100% agree about a family mode.

**Comment by TravelinMax** [comment]
I was thinking a widget, but I guess shortcut support accomplishes the same thing. 👍

**Comment by amiantos** [comment]
Bonus note: the iOS version will be an offshoot of the Mac version. So if you buy the Mac version from the App Store you'll automatically get the iOS version for free when it comes out.

**Comment by asherman07** [comment]
There is an app called PHRemote that is able to disable pihole but only works with one pihole and has some other cool features might be something to look at to get ideas!

**Comment by amiantos** [comment]
@asherman07 thanks for the tip! I dunno if you use the subreddit but recently I [posted a little video of PiBar for iOS](https://www.reddit.com/r/PiBar/comments/ih683y/work_on_pibar_for_ios_continues_example_of_theme/). Lemme know what you think!

**Comment by asherman07** [comment]
I just saw this now! this looks way better than the app I use now! Can't wait for the release!

**Comment by TravelinMax** [comment]
Looks great, can't wait!
